theme/entity_plus.theme.inc

```diff
@@ -139,7 +139,13 @@ function entity_plus_property_default_render_value_by_type(EntityMetadataWrapper
  */
 function template_preprocess_entity_plus(&$variables) {
   $variables['view_mode'] = $variables['elements']['#view_mode'];
-  $entity_type = $variables['elements']['#entity_plus_type'];
+  $entity_type = $variables['elements']['#entity_type'];
```

Comment:
To be on the safe side, should we check that `#entity_type` is set? There may be cases where a custom module returns a render array with `#theme => 'entity_plus'` but still uses the old `#entity_plus_type` instead of `#entity_type`.
 
So, a fail-save check may look like:

```php
$entity_type = !empty($variables['elements']['#entity_type']) ? $variables['elements']['#entity_type'] : $variables['elements']['#entity_plus_type'];
```